symLogging.py

```diff
@@ -47,21 +47,26 @@ def SetLoggingOptions(logOptions):
   logLevel = getattr(logging, logOptions.get("logLevel", "INFO"))
   gLog.setLevel(logLevel)
 
-def doLog(dbgLevel, string):
+def doLog(dbgLevel, string, remoteIp):
   pid = os.getpid()
-  gLog.log(dbgLevel, "%d\t%s", pid, string)
-
-def LogDebug(string):
+  gLog.log(
+    dbgLevel,
+    "%d\t%s%s",
+    pid,
+    remoteIp + ' ' if remoteIp else '',
+    string)
```

Comment:
hmm, maybe add it as the last field and use a unique format to make it easier to filter on with grep? so maybe

gLog.log(dbgLevel, "%d\t%s%s", pid, string, "IP=" + remoteIP + " " if remoteIP else "")